charts/dex/values.yaml

```diff
@@ -120,6 +120,8 @@ ingress:
   #  - secretName: chart-example-tls
   #    hosts:
   #      - chart-example.local
+  # ingressClassName support
```

Comment:
Could you add more info about what this parameter does (maybe with the link to Kubernetes documentation)?